package.json

```diff
@@ -14,7 +14,7 @@
 		"build": "lerna run build --stream",
 		"format": "yarn lint --fix",
 		"lerna:deploy": "yarn build && lerna version --conventional-commits && lerna publish from-git",
-		"lint": "tsc --noEmit && eslint --ext js,ts,tsx .",
+		"lint": "eslint --ext js,ts,tsx .",
```

Comment:
Currently there are typescript errors in the generated code. Removing ts compile from the lint script for now but we may want to add back when we have a solution.